testserv.c

```diff
@@ -166,16 +196,43 @@ mustforksrv(void)
 
     int port = ntohs(addr.sin_port);
     srvpid = fork();
-    if (srvpid < 0) {
-        twarn("fork");
+    if (srvpid > 0) {
```

Comment:
Here the original code contained the check for srvpid < 0:
```
if (srvpid < 0) {
   twarn("fork");	
   exit(1);
}
```
The same check should happen in mustforksrv_unix

Reply:
Oops :-(